soc/arm/microchip_mec/mec172x/Kconfig.defconfig.mec172xnsz

```diff
@@ -47,4 +47,8 @@ config PECI_XEC
 	default y
 	depends on PECI
 
+config SPI_XEC_QMSPI_V2
+	default n
+	depends on SPI
+
```

Comment:
What's going on here?

Reply:
We have two SPI drivers for MEC172x, the latest version 2 was added to work with the Zephyr NOR driver.
So the application can enable only one of the two XEC SPI drivers, CONFIG_SPI_XEC_QMSPI_LDMA or CONFIG_SPI_XEC_QMSPI_V2.
We cannot turn on both (qmspi_v2) and qmspi_ldma at the same time.

Reply:
Should this be a `choice` then?

Reply:
yes, will consider `choice`

Reply:
updated with choice